docs/book/advanced-guide/practical/code-repositories.md

```diff
@@ -0,0 +1,195 @@
+---
+description: How to track the code of your pipelines
+---
+
+## What is a code repository
+
```

Comment:
Again needs a visualization of what is different using a code repository and what happens when not using it